packages/babel-parser/src/parser/statement.js

```diff
@@ -135,7 +135,7 @@ function babel7CompatTokens(tokens) {
             // ends with '`'
             templateElementEnd = end - 1;
             templateElementEndLoc = createPositionWithColumnOffset(loc.end, -1);
-            templateValue = value.slice(1, -1);
+            templateValue = value === null ? null : value.slice(1, -1);
```

Comment:
Nit: we can use `value?.slice`

Reply:
I don't think so. We need returning `null` when `value` is `null`.